apps/docs/src/app/platform/component-docs/platform-forms/platform-form-generator/platform-form-generator-examples/platform-form-generator-custom-component-example.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component } from '@angular/core';
+
+import {
+    dynamicFormFieldProvider,
+    dynamicFormGroupChildProvider,
+    BaseDynamicFormGeneratorControl,
+    FormGeneratorService,
+    DynamicFormItem
+} from '@fundamental-ngx/platform';
```

Comment:
please use the appropriate imports path

Reply:
Platform does not support deep imports yet